theano/sandbox/gpuarray/basic_ops.py

```diff
@@ -145,11 +145,18 @@ def _generate_kernel_vars(self, k):
         return """static GpuKernel %(kname)s;""" % dict(kname=k.objvar)
 
     def c_support_code_apply(self, node, name):
+        ceil_intdiv = """
+        template <typename T>
+        static T ceil_intdiv(T a, T b)
+        {
+            return (a/b) + ((a % b) ? 1: 0);
+        }
+        """
```

Comment:
This could go in c_support_code() since nothing is specific to the apply node.

Also, because I'm curious, why do you need this?